src/AccessToken.php

```diff
@@ -411,7 +416,11 @@ private function loadPhpsecPublicKey(string $modulus, string $exponent): string
                     $exponent
                 ]), 256),
             ]);
-            return $key->getPublicKey();
+            $formattedPublicKey = $key->getPublicKey();
+            if (is_string($formattedPublicKey)) {
+                return $formattedPublicKey;
+            }
+            throw new TypeError('Failed to initialize the key');
         }
```

Comment:
We can actually remove this first `if` block since this is from an old phpseclib version, which is not used anymore due to our [composer.json](https://github.com/googleapis/google-auth-library-php/blob/b8e4dad3fd2f96bb76699bf9a22e40c1b1d2f471/composer.json#L25). Thoughts?
```
    "phpseclib/phpseclib": "^3.0",
```

Reply:
Addressed in https://github.com/googleapis/google-auth-library-php/pull/518